testsuite/tests/effects-api/cross_capsule.ml

```diff
@@ -0,0 +1,47 @@
+(* TEST
+ include stdlib_alpha;
+ runtime5;
+ { bytecode; }
+ { native; }
+*)
+
+[@@@alert "-unsafe_parallelism"]
+
+open Stdlib_alpha
+
+type 'a op = 
+  | Set : int -> unit op
+  | Get : int op
+
+module Eff = Effect.Make(struct
+  type 'a t = 'a op
+end)
+
+let store = Atomic.Safe.make 0
+
+let rec handle = function
+  | Eff.Result.Contended.Value v -> v
+  | Exception e -> raise e
+  | Operation (Set n, k) ->
+    Atomic.Safe.set store n;
```

Comment:
Why is this using `Atomic.t` rather than just `ref`? I don't think you are relying on `store` itself being `contended` anywhere.

Reply:
Fixed.